README.md

```diff
@@ -25,87 +25,81 @@
     <img src="./assets/flat-showcase-en.png">
 </div>
 
-## Try it now
+## Features
 
-- [Start using Flat Web][flat-web]
-- [Download artifact][flat-homepage]
-- [Flat Components Storybook][flat-storybook]
+-   Real-time interaction
+    -   Multifunctional interactive whiteboard
+    -   Real-time video/audio chat (RTC)
+    -   Real-time messaging (RTM)
+-   Login via
+    -   WeChat
+    -   GitHub
+-   Classroom management
+    -   Join, create and schedule classrooms
+    -   Support periodic rooms
+-   Classroom recording and replaying
+    -   Whiteboard replaying
+    -   Cloud recording for video and audio
+    -   Messaging replaying
+-   Cloud Storage for multi-media courseware
+-   Screen sharing
 
-## Related Projects
+## Quickstart
```

Comment:
QuickStart
or
Quick Start

Reply:
`Quickstart` is the correct spell.

Reply:
@yamasite 
please add `Quickstart` to https://github.com/netless-io/flat/blob/d6c668ab4fdd81b0662beee5b668d0a39604d058/cspell.config.js#L18-L19